src/main/resources/db/migration/V4__add_revenue_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE Revenues(
```

Comment:
If you create 3 migration scripts at the same time (they are merged together in master which is also your case here) you should give them the same version. This means your files should have the following names: V4_1__add_revenue_table, V4_2__add_location_to_order_details, V4_3__add_foreign_key_.... In the way you named them, flyway will detect that you have 3 changes of the application's version (3 migrations).